protocols/aavegotchi.eth/index.json

```diff
@@ -0,0 +1,33 @@
+{ 
+	"cname": "aavegotchi",
+	"name": "Aavegotchi",
+	"description": "",
+	"path": "aavegotchi.eth",
+	"previousPaths": [],
+	"folder": "aavegotchi",
+	"type": "snapshot",
+	"suffix": "GHST",
+	"coinGeckoPriceString": "",
```

Comment:
New API provides us with [token market prices](https://github.com/boardroom-inc/boardroom-platform/blob/main/packages/boardroom-api/src/protocols.ts#L25-L28) but we're still using `coinGeckoPriceString` in the [claims modal](https://github.com/boardroom-inc/hub-ui/blob/master/src/features/Claim/ClaimsModal.tsx#L135), so we should either define it here or update that component to source token price from the API.

Reply:
Sure, sorry it wasnt done, i was just testing about the keys I don't even need in the first place. like forum etc. hence I haven't requested any reviews too. Gimme 5. I will go through this in a bit

Reply:
Ahh my bad, makes sense 👍

Reply:
> update that component to source token price from the API.

i think I can do that, but lets do that in a separate PR. i was thinking about removing keys and use API/SDK as single source of truth for even like snapshotSpacename etc.  i will create some tasks around it.

Reply:
Dig it.

Reply:
Created tech-debt tasks in QA board.